### DISCUSSION TITLE
NRF52840  how to read data from internal themperature sensor

### DISCUSSION THREAD

**Discussion by zzolax** [zephyrproject-rtos/zephyr#58118] [Q&A]
Hello,
I tried example samples\sensor\termometer, added CONFIG_TEMP_NRF5=y in prj.conf but have
error: no temp device

**Comment by ShahinHaque97-CT** [comment]
This may be of some help. In the nRF Connect SDK, 
https://github.com/nrfconnect/sdk-nrf/blob/main/samples/bluetooth/mesh/sensor_server/src/model_handler.c

This example has a device node for internal die temperature.
